js/genkit/package.json

```diff
@@ -0,0 +1,185 @@
+{
+  "name": "genkit",
+  "description": "Genkit AI framework",
```

Comment:
Is this shown somewhere publicly? We might want to expand on this (in another PR is fine).

Reply:
not sure actually... maybe.... yeah